nndownload/nndownload.py

```diff
@@ -1911,23 +1912,186 @@ def download_thumbnail(session: requests.Session, filename: AnyStr, template_par
 
     output("Finished downloading thumbnail for {0}.\n".format(template_params["id"]), logging.INFO)
 
-
-def download_comments(session: requests.Session, filename: AnyStr, template_params: dict):
+def get_niconico_timestamp(input_time=None):
```

Comment:
Doesn't really seem like it's specific to Nico, so should it just be `get_unix_timestamp`? If so, it should be placed under `## Utility methods`